examples/38_cookies/src/server-entry.tsx

```diff
@@ -13,13 +11,7 @@ export default defineEntries({
     const data = getContextData() as { count?: number };
     data.count = (data.count || 0) + 1;
     const items = JSON.parse(
-      await fsPromises.readFile(
-        path.join(
-          path.dirname(fileURLToPath(import.meta.url)),
-          '../private/items.json',
-        ),
-        'utf8',
-      ),
+      await fsPromises.readFile('./private/items.json', 'utf8'),
```

Comment:
Does this make to read file from "cwd" instead of the relative path from this file?

Reply:
Sounds fair.

I wonder if it works with cloud deploys.

> where this piece of code ends up in `dist` is not so deterministic

I actually tried to keep some module structure with `preserveModuleDirs` before, which no longer is there. So, we can't technically rely on `import.meta.url`. Hmm.

Reply:
Yes, for node serverless deploy (vercel, netlify, etc...), private dir is copied to the server root so it should work, but not on cf or other edges.